tests/test_core_files.py

```diff
@@ -27,6 +28,13 @@ def make_stat(mode=33200, uid=0, gid=0):
             mode = 0660
             owner = root
             group = root
+
+       Cheat sheet equivalents:
+           0600 = 33152
+           0640 = 33184
+           0644 = 33188
+           0660 = 33200
+           0666 = 33206
```

Comment:
You have no idea how thankful I am for these lines! I'm sure I would have forgotten how to map things over the night :)

Reply:
:-)

thanks for the review.